src/ImageSharp/Formats/ImageExtensions.Save.cs

```diff
@@ -535,5 +536,108 @@ public static void SaveAsTga(this Image source, Stream stream, TgaEncoder encode
                 encoder ?? source.GetConfiguration().ImageFormatsManager.FindEncoder(TgaFormat.Instance),
                 cancellationToken);
 
+        /// <summary>
+        /// EXPERIMENTAL! Saves the image to the given stream with the Tiff format.
```

Comment:
Let's remove all comments that describe the format as experimental.

Reply:
Changed with fa6401c0